### PR TITLE
Build with Xcode 13.2.1 for release

### PR DESCRIPTION
**Pull request by luispadron**
After merging https://github.com/MobileNativeFoundation/XCLogParser/pull/177 we'll need to bump Xcode to pull in Swift 5.5+

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     name: Add macOS binaries to release
     runs-on: macOS-latest
     env:
-      XCODE_VERSION: ${{ '12.5' }}
+      XCODE_VERSION: ${{ '14.1' }}
```
Comment (luispadron) [inline .github/workflows/release.yaml:11]:
```suggestion
      XCODE_VERSION: ${{ '13.2.1' }}
```

**Comment by BalestraPatrick** [comment]
@luispadron Were there failures with 14.1 or?

**Comment by luispadron** [comment]
@BalestraPatrick i didn't let the job finish since I realized the rest of the jobs are using 13.2.1 so didn't want the mismatch. 

I can update them all to Xcode 14.1 if you prefer though!

**Comment by BalestraPatrick** [comment]
That's fine, we can bump to Xcode 14.1 in a separate PR.

**Review by BalestraPatrick — APPROVED** [review]

**Comment by luispadron** [comment]
When ya'll have a chance, could we tag a new release 🙏🏼

**Comment by BalestraPatrick** [comment]
It's done! https://github.com/spotify/XCMetrics/releases/tag/0.0.11

**Pull request merged**